src/mRides-server/Controllers/UserController.cs

```diff
@@ -39,7 +39,12 @@ public object getUser(int id)
         {
             return _userCatalog.get(id);
         }
+        [HttpPost]
+        public Boolean registerFcmToken([FromHeader]string id, [FromBody]dynamic sentObj)
+        {
+            int userid = Convert.ToInt32(id);
 
```

Comment:
The return value is a Boolean type, but here the function is not returning anything?